src/main/java/com/ncc/aif/AIFUtils.java

```diff
@@ -160,7 +160,10 @@ public static Resource makeRelation(Model model, String relationUri, Resource sy
      * @param system           The system object for the system which created the specified relation
      * @param confidence       If non-null, the confidence with which to mark the specified relation
      * @return The created relation resource
+     * @deprecated AIF no longer represents relations as simply a connection between two entities,
```

Comment:
```suggestion
     * @deprecated This method doesn't allow the user access to the blank nodes created for type and argument assertions. Use {@link #makeRelation(Model, String, Resource)} instead
```

Reply:
Much better, thanks.  I also noticed it uses the same confidence for all assertions, which might not be what you want.